test/serveable_file_test.rb

```diff
@@ -14,6 +14,7 @@ def setup
 
       def teardown
         revert_to_original_working_directory
+        Rack::Zippy.reset_static_extensions
```

Comment:
I *think* the reset won't work as intended as the original default array is changed by `Rack::Zippy.static_extensions << 'csv'`.